howfairis/ReadmeFormat.py

```diff
@@ -7,3 +7,4 @@
 class ReadmeFormat(Enum):
     MARKDOWN = auto()
     RESTRUCTUREDTEXT = auto()
+    HTML = auto()
```

Comment:
I know this is still a WIP, but including HTML here is maybe not a good idea, since there is no file named `README.html` in the repo tree. Just a thought.

Reply:
Thanks for thinking with me.

I wanted to write out rst without the comments
Sadly docutils does not have a rst writer. 

Tried to find one in other packages:

* [rst2rst](https://github.com/benoitbryon/rst2rst) does but is Python2 .
* https://github.com/sphinx-contrib/restbuilder/blob/master/sphinxcontrib/writers/rst.py is tied with Sphinx
*

Reply:
got it working with https://github.com/dzhu/rstfmt, but it is hacky and drags in a lot of dependencies.

Reply:
Still want to try to get the comment position using a visitor

Reply:
Yeah, node.line is line number of original text where node appears. As comments always start at beginning of line ignore those lines.